packages/critters/src/util.js

```diff
@@ -38,3 +39,8 @@ export function createLogger(logLevel) {
     return logger;
   }, {});
 }
+
+export function isSubpath(basePath, currentPath) {
+  const relative = path.relative(basePath, currentPath);
+  return relative && !relative.startsWith('..') && !path.isAbsolute(relative);
```

Comment:
`path.relative` does not return an absolute path. 

I believe the below should cover all the cases.

```suggestion
  return !path.relative(basePath, currentPath).startsWith('..')
```